python/python_direct/ops.cpp

```diff
@@ -1623,6 +1641,104 @@ TensorView
 )")
 }
 
+void bindScanOps(py::module_& ops) {
+  // cumsum (prefix sum) along a dimension
+  NVFUSER_DIRECT_BINDING_SCAN_OP(
+      "cumsum",
+      scan,
+      BinaryOpType::Add,
+      R"(
+Scan (prefix sum) along a dimension.
```

Comment:
The docstring should be updated for `cumsum`, `cumprod`, `cummin`, and `cummax`.